pypeit/core/findobj_skymask.py

```diff
@@ -1413,8 +1416,11 @@ def objfind_QA(spat_peaks, snr_peaks, spat_vector, snr_vector, snr_thresh, qa_ti
     fig = plt.gcf()
     if show:
         plt.show()
+    # Write to disk?
     if objfindQA_filename is not None:
-        fig.savefig(objfindQA_filename, dpi=400)
+        qafile = Path(objfindQA_filename).absolute()
+        qafile.parent.mkdir(parents=True, exists_ok=True)
```

Comment:
```suggestion
        qafile.parent.mkdir(parents=True, exist_ok=True)
```

Reply:
I committed this change to get the dev-suite to run